pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -277,16 +268,7 @@ func RunNodeConfigBootstrap(templateDir string, featureGateAccess featuregates.F
 	if nodeConfig == nil {
 		return nil, fmt.Errorf("nodes.config.openshift.io resource not found")
 	}
-	// explicitly setting the cgroupMode to "v2" and also updating the config node's spec if found empty
-	// This helps in updating the cgroupMode on all the worker nodes if they still have cgroupsv1 (Ex: RHEL8 workers)
-	if nodeConfig.Spec.CgroupMode == osev1.CgroupModeEmpty {
```

Comment:
Could you remind me why this is being removed?

Reply:
We are not going to update the API object, which causes a reconcile event to happen in the MCO. CgroupModeEmpty will default to cgroupsv2... Older installations will have v1 set in cgroupmode already (forcing cgroupv1).